lib/testScenario/bodyTransformer.ts

```diff
@@ -101,9 +101,9 @@ export class BodyTransformer {
 
     if (typeof dst === "object" && typeof src === "object") {
       const result: any = { ...dst };
```

Comment:
what if dst is null and src is not null ?

Reply:
It will not entry this code. So it's safe

Reply:
could you add comment here